src/main/java/com/salesforce/dataloader/controller/Controller.java

```diff
@@ -86,6 +86,7 @@
     private static final String LOG_CONF_OVERRIDE = "log-conf.xml";
 
     private static boolean isLogInitialized = false; // make sure log is initialized only once
+    private static boolean areStaticVarsInitialized = false; // make sure log is initialized only once
```

Comment:
Minor, usually "isSomething" although the context is plural

Reply:
Exactly!